src/backend/sgx/attestation.rs

```diff
@@ -298,8 +298,8 @@ pub fn get_quote(report: &[u8], akid: Vec<u8>, out_buf: &mut [u8]) -> Result<usi
 mod tests {
     use super::*;
 
```

Comment:
looks like this commit is already in `main`: https://github.com/enarx/enarx/commit/cc0a794e40a96481a588aaebc3ea1a57cd20a523, please rebase on latest `main`

Reply:
Fixed